api/sys_plugins.go

```diff
@@ -36,6 +36,8 @@ type ListPluginsResponse struct {
 type PluginDetails struct {
 	Type              string `json:"type"`
 	Name              string `json:"name"`
+	OCIImage          string `json:"oci_image" mapstructure:"oci_image"`
```

Comment:
I looked into earlier in the call stack that constructs `PluginDetails`. Looks like `oci_image` and `runtime` don't need to be added here since`detailed` only contains the versioned plugins https://github.com/hashicorp/vault/blob/main/vault/logical_system.go#L463-L474.